cli/Tests/TuistKitTests/Services/Inspect/InspectRedundantImportsServiceTests.swift

```diff
@@ -322,51 +322,68 @@ final class LintRedundantImportsServiceTests: TuistUnitTestCase {
         try await subject.run(path: path.pathString)
     }
 
-    func test_run_doesntThrowAnyErrorsWithAppWatchExtensionsSetWithStickerPackExtension_when_thereAreNoIssues() async throws {
+    func test_run_doesntThrowAnyErrorsWithAppWithAppWatch_when_thereAreNoIssues() async throws {
```

Comment:
This test was wrong. Was added when added supporting for app to depend on watchApp, but was doing the wrong check